OpenRA.Mods.TS/Traits/Render/WithVoxelWalkerBody.cs

```diff
@@ -24,6 +24,8 @@ namespace OpenRA.Mods.TS.Traits.Render
 {
 	public class WithVoxelWalkerBodyInfo : ITraitInfo, IRenderActorPreviewVoxelsInfo,  Requires<RenderVoxelsInfo>, Requires<IMoveInfo>, Requires<IFacingInfo>
 	{
+		public readonly string Sequence = "idle";
```

Comment:
If we are going to unhardcode these then we really should have a lint check for them.